### PR TITLE
task/frontend-input-image-preview-and-result

### PR DESCRIPTION
**Pull request by rikardomarenzzi**
The result of the aws service is now connected to the query of the TMDB database.

### CODE REVIEW

File: frontend/src/previewPage/PreviewPage.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+      <div>{awsResult}</div>
+      </>
+  )
+
+  function onCancel() {
+    history.goBack();
+  }
+
+  function recognizeCelebrity() {
+    console.log(inputImage);
```
Comment (FabianKeck) [inline frontend/src/previewPage/PreviewPage.js:25]: Do you need this?

**Review by FabianKeck — APPROVED** [review]

File: frontend/src/previewPage/PreviewPage.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  const history = useHistory();
+
+  return (
+    <>
+      <div>PreviewPage</div>
+      <img src={inputImageUrl} alt="upload" height="50%" width="50%"/>
+      <div>Wollten Sie dies Person fotografieren?</div>
+      <button onClick={recognizeCelebrity}>Ja</button>
+      <button onClick={onCancel}>Nein</button>
+      <div>{awsResult}</div>
```
Comment (fabianschmauder) [inline frontend/src/previewPage/PreviewPage.js:16]: i would not name your variables some thing with "aws". When you switch to  another api you have to rename all variables

**Review by fabianschmauder — COMMENTED** [review]
looks good. maybe choose different names for variables

File: frontend/src/previewPage/PreviewPage.js (hunk 1 above)
Comment (rikardomarenzzi) [reply frontend/src/previewPage/PreviewPage.js:25]: will remove it in next PR

File: frontend/src/previewPage/PreviewPage.js (hunk 2 above)
Comment (rikardomarenzzi) [reply frontend/src/previewPage/PreviewPage.js:16]: Good point, I will change that

**Pull request merged**